oml/utils/audios.py

```diff
@@ -0,0 +1,125 @@
+import base64
+from io import BytesIO
+from typing import Union
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from PIL import Image
+from torch import FloatTensor
+
+from oml.const import BLACK, DEFAULT_SAMPLE_RATE, TColor
+
+
+def default_spec_repr_func(audio: FloatTensor) -> FloatTensor:
+    """
+    Generate a default spectral representation (log-scaled MelSpec) from an audio signal.
+    Used primarily for visualization.
+
+    Args:
+        audio: The input audio tensor.
+
+    Returns:
+        The spectral representation of the input audio tensor.
+    """
+    from torchaudio.transforms import MelSpectrogram
+
+    melspectrogram = MelSpectrogram(sample_rate=DEFAULT_SAMPLE_RATE, n_fft=2048, hop_length=512, n_mels=128)
+    melspec = melspectrogram(audio)
+    log_melspec = torch.log1p(melspec).squeeze(0)
+    return log_melspec
+
+
+def _visualize_audio(
+    spec_repr: FloatTensor, color: TColor = BLACK, draw_bbox: bool = True, return_b64: bool = False
+) -> Union[np.ndarray, str]:
+    """
+    Internal function to visualize an audio spectrogram.
+
+    Args:
+        spec_repr: The spectrogram representation as a FloatTensor.
+        color: The color of the bounding box.
+        draw_bbox: Whether to draw a bounding box around the spectrogram.
+        return_b64: Whether to return the image as a base64 string.
+
+    Returns:
+        If return_b64 is False, returns the image as a NumPy array.
```

Comment:
please add `` everywhere it's needed in the docs

Reply:
Done